Assets/Photon Unity Networking/Resources/PhotonServerSettings.asset

```diff
@@ -15,14 +15,14 @@ MonoBehaviour:
   AppID: 
   VoiceAppID: 
   ChatAppID: 
-  HostType: 2
```

Comment:
Changing the HostType & JoinLobby setting breaks multiplayer, undo these changes